cvat-ui/src/components/task-page/details.tsx

```diff
@@ -92,8 +94,9 @@ class DetailsComponent extends React.PureComponent<Props, State> {
     }
 
     private renderTaskName(): JSX.Element {
-        const { name } = this.state;
+        const { name, consensusJobsPerSegment } = this.state;
         const { task: taskInstance, onUpdateTask } = this.props;
+        const taskName = name + (consensusJobsPerSegment > 0 ? ' (Consensus Based Annotation)' : '');
```

Comment:
I suggest that the task name should not be modified, and an icon or a tag should be shown instead. Take for example the "Ground Truth" label on GT jobs.